todo.js

```diff
@@ -72,28 +82,46 @@ function taskStatus(e) {
     for (const task of createdTasks) {
         if (task.description === e.currentTarget.nextSibling.value) {
             task.complete = e.currentTarget.checked;
+            if (task.complete === true) {
+                task.date = completedDateGenerator();
+                e.currentTarget.parentElement.children[0].textContent = task.date;
+            } else {
+                task.date = 'COMPLETED DATE: NOT READY YET';
```

Comment:
Add `completed date ` to `completeTaskDateDom` function instead of re-creating this string every time